github/compliance/report.py

```diff
@@ -596,23 +596,14 @@ def _target_sprint(
     return target_sprint
 
 
-def _latest_processing_date(
```

Comment:
was this function unused? if not, this commit (eccdf62ac9ec9d0079cf726eeec232a5af1a19b5) will lead to a broken state

Reply:
Yes, it was unused across cc-utils and our other repos.